.github/workflows/pkgci.yml

```diff
@@ -37,17 +37,19 @@ jobs:
     with:
       package_version: 0.dev1
 
-  regression_test_cpu:
-    name: Regression Test CPU
```

Comment:
While we sort out the GPU runner issues, it would be good to get this CPU job back to using a standard cloud CPU runner instead of one of our self-hosted GPU runners -- at least for small tests that don't need a persistent cache of real model weight files.